packit_service/worker/github_handlers.py

```diff
@@ -241,7 +241,9 @@ def handle_pull_request(self):
 
         collaborators = self.project.who_can_merge_pr()
         r = BuildStatusReporter(self.project, self.event.commit_sha)
-        if self.event.github_login not in collaborators:
+        if (self.event.github_login not in collaborators) and (
+            self.event.github_login not in self.config.admins
```

Comment:
`collaborators` is already a `set` so you can:
```python
if self.event.github_login not in collaborators | set(self.config.admins):
```
or if you define `config.admins` as `set` then even more simple:
```python
if self.event.github_login not in collaborators | self.config.admins:
```